spec/label_spec.rb

```diff
@@ -0,0 +1,14 @@
+require './classes/item'
+require './classes/label'
+
+describe Label do
```

Comment:
I think it would be better to have unit tests for the class's instance attributes and methods as well so that we can be more confident about our tests. :+1:

Reply:
👍